doc/modules/mixture.rst

```diff
@@ -211,49 +211,75 @@ Cons
 
 .. _dpgmm:
 
-DPGMM: Infinite Gaussian mixtures
-=================================
+Dirichlet Process: Infinite Gaussian mixture
+============================================
+
+The :class:`DirichletGaussianMixture` class implements a variant of the
+Gaussian mixture model with a variable (but bounded) number of components using
+the Dirichlet Process.
 
-The :class:`DPGMM` object implements a variant of the Gaussian mixture
-model with a variable (but bounded) number of components using the
-Dirichlet Process.
 This class doesn't require the user to choose the number of
 components, and at the expense of extra computational time the user
 only needs to specify a loose upper bound on this number and a
 concentration parameter.
 
-.. |plot_gmm| image:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_001.png
+
+The examples bellow compare Gaussian mixture models with fixed number of
+components, to the Dirichlet Gaussian mixture models.
+
+.. figure:: ../auto_examples/mixture/images/sphx_glr_plot_gmm_001.png
    :target: ../auto_examples/mixture/plot_gmm.html
-   :scale: 48%
+   :align: center
+   :scale: 70%
+
+Here, a classical Gaussian mixture is fitted with 5 components on a dataset
+composed of 2 clusters. We can see that the Dirichlet Gaussian mixture is able
+to limit itself to only 2 components whereas the Gaussian mixture fits the data
+fit with too many components. Note that with very little observations, the
```

Comment:
=> whereas the Gaussian mixture fits the data with a fixed number of components that has to be set a priori by the user. In this case the user has select `n_components=5` which does not match the true generative distribution of this toy dataset.